lib/e164/lookuptable.rb

```diff
@@ -1,646 +1,690 @@
-# This file is automatically generated by `rake rebuild`
+# frozen_string_literal: true
+
+# USED TO BE -> This file is automatically generated by `rake rebuild`,
+#   library is no longer maintained
+
+# this file is manually edited by pulling down the NAPA database
+#   and editing this file:
+
+# https://nationalnanpa.com/reports/area_code_relief_planning.html
+
+# this only updated the US country codes.  Everything else is < 2015.
+# We generally only care if it's a USA phone number
+
+# rubocop:disable Metrics/ModuleLength
+# rubocop:disable Naming/ConstantName
+# rubocop:disable Style/MutableConstant
+# rubocop:disable Style/WordArray
```

Comment:
Don't disable any of these cops except for `ModuleLength`. You can run `rubocop -a` to autofix the `MutableConstant` and `WordArray` ones, and the `ConstantName` one will just require a replacement of `LookupTable` with `LOOKUP_TABLE`.

Reply:
I don't want to mess with this now because of comparing the previous tables.  

I tried the rubocop -a stuff and it actually broke a bunch of things on my first try.  I wasn't super happy with it